book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] :rentals should also be in the getter and setter so that it could be read and set since we will be adding(setter) data based on some available(getter) data.